src/webgpu/util/compare.ts

```diff
@@ -174,3 +181,18 @@ export function ulpComparator(x: number, target: Scalar, n: (x: number) => numbe
     };
   };
 }
+
+export function intervalComparator(i: FPInterval): Comparator {
+  const match = intervalMatch(i);
+  return (got, _) => {
+    const cmp = compare(got, f64(0.0), match);
```

Comment:
This invocation actually ends up ignoring the 2nd arg (f64(0.0))?   Because the comparisons are actually being done by the matcher.  That's......a little too cute for production.   OR add a prominent comment.  :-)

Reply:
I agree this is too cute, I intend to take a pass at reworking how the whole runner/comparator thing is done once we are happy with how the intervals side of things is expressed.